### DISCUSSION TITLE
with insiders, how to run locally a replica of the doc with the website?

### DISCUSSION THREAD

**Discussion by Y0lan** [squidfunk/mkdocs-material#7623] [Q&A]
I love everything about the web and the doc and how it is designed and laid out, so I paid for access to insider build to be able to do a complete fork and run it locally with the web and accessing the docs as well.
Any idea how to do this?

**Comment by kamilkrzyskow** [comment]
Hello @Y0lan,
if I understood you correctly you want to run the Material for MkDocs documentation page locally on your PC, while using the Insiders version?

You can follow the instructions laid out here:
- https://squidfunk.github.io/mkdocs-material/customization/#environment-setup-insiders
this will allow you to see the site locally.

To make it accessible on the web you need to deploy it, instructions can be found here:
- https://squidfunk.github.io/mkdocs-material/publishing-your-site/#with-github-actions-insiders
- https://github.com/squidfunk/mkdocs-material/discussions/7336

Let us know if something is not clear :v:

**Reply by Y0lan** [reply]
Hi, thank you for the reply.

Yes I installed it correctly and deployed it live, it shows the website perfectly well. 

Is there a way to simply modify the current web?

I tried changing text and some img in src/overrides but the change did not show on the web.
After some digging, I found out about a generated material folder. How to rebuild it?
Any suggestion ? Thanks!

**Reply by squidfunk** [reply]
This is all answered in [our documentation on customization](https://squidfunk.github.io/mkdocs-material/customization/), so I recommend reading it if you want to override parts of the theme.
